src/problem/MFEMProblem.C

```diff
@@ -93,4 +118,105 @@ MFEMProblem::addMaterial(const std::string & kernel_name,
     }
     _mat_map.materials.push_back(mat);
   }
+
+}
+
+
+void MFEMProblem::addAuxVariable(const std::string& var_type, 
+                                 const std::string& var_name,
+                                 InputParameters& parameters)
+{
+  //Standard Moose implementation
+  auto var_order = Utility::string_to_enum<Order>(parameters.get<MooseEnum>("order"));
+  std::string var_family = parameters.get<MooseEnum>("family");
+  auto fe_type = FEType(var_order, Utility::string_to_enum<FEFamily>(var_family));
+
+  if (duplicateVariableCheck(var_name, fe_type, /* is_aux = */ true))
+  {
+    return;
+  }
+    
+  parameters.set<FEProblemBase *>("_fe_problem_base") = this;
+  parameters.set<Moose::VarKindType>("_var_kind") = Moose::VarKindType::VAR_AUXILIARY;  
+  _aux->addVariable(var_type, var_name, parameters);
+
+  if (_displaced_problem)
+  {
+    _displaced_problem->addAuxVariable(var_type, var_name, parameters);
+  }
+  //End of standard implementation
```

Comment:
If this is from the base class - worth just calling the base class method explicitly? Would remove duplication of this logic block, and better insulate against changes in fe_problem on the MOOSE side